incubator/acs-engine-autoscaler/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: Scales worker nodes within autoscaling groups.
```

Comment:
This chart can be in `stable/`. incubator is required only if you're using experimental Kubernetes functionality: alpha APIs, for example.